internal/controller/controller.go

```diff
@@ -300,6 +316,10 @@ func (c *Controller) createMeshService(service *corev1.Service) error {
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      meshServiceName,
 				Namespace: c.meshNamespace,
+				Labels: map[string]string{
+					"app":       "maesh",
+					"component": "maesh-svc",
```

Comment:
Is this component label required if we already are only using the app label?

Also, could we not have these labels generated by a similar function referenced in:
https://github.com/containous/maesh/pull/348/files#diff-0d5beddb6a9b5c9dbe4fd5fe543b2471R235

So that a change to one would reflect in the other?

Reply:
> Is this component label required if we already are only using the app label?

Nope,

> Also, could we not have these labels generated by a similar function referenced...

Will see what I can do.

Reply:
> Also, could we not have these labels generated by a similar function referenced...

I actually removed the need of having a label for "app: maesh" label so no need to factorize this here, right ?